en/releases/daily_builds.md

```diff
@@ -1,20 +1,28 @@
 # Daily Builds
 
-Daily Builds of QGroundControl are also available for use. These builds have less testing than Stable builds but will have the absolute latest set of [new features](../releases/daily_build_new_features.md) available. Use at your own risk!
+Daily Builds of *QGroundControl* have the absolute latest set of [new features](../releases/daily_build_new_features.md). 
+
+> **Warning** Daily Builds are less tested than stable builds. Use at your own risk!
+
+These can be downloaded from the links below:
 
 * [Windows](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl-installer.exe)
 * [OS X](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.dmg)
 * Linux
```

Comment:
@DonLakeFlyer Would I be correct that for Linux you either install the AppImage, OR you install the Compressed Archive AND the Qt libraries (ie you only need the additional packages IF you use the archive)?